polling_stations/templates/home.html

```diff
@@ -17,7 +17,7 @@
       <label class="card_header">{% trans "Enter Your Postcode" %}
         <input id="id_postcode" name="postcode" type="text" aria-describedby="postcode_help_text" autofocus >
       </label>
-      <p class="help-text" id="postcode_help_text">{% trans 'e.g. <a href="postcode/CF105AJ/">CF10 5AJ</a>' %}</p>
+      <p class="help-text" id="postcode_help_text">{% trans 'e.g. <a href="/postcode/CF105AJ/">CF10 5AJ</a>' %}</p>
```

Comment:
This will break when in embed mode (see http://pollingstations.democracyclub.org.uk/embed/), so might need to be replaced by a `{% url %}` tag, and to be tested at that URL prefix.